flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/HttpRequestHandler.java

```diff
@@ -99,7 +110,43 @@ public void channelRead0(ChannelHandlerContext ctx, HttpObject msg) {
 					currentDecoder.destroy();
 					currentDecoder = null;
 				}
-				
+
+				if(secureCookie != null) {
```

Comment:
The secure cookie value could be auto-populated (Yarn) or user-provided but will be persisted in the in-memory Flink configuration instance which is passed to the web layer during bootstrap. Should the user decide to torn security off, then we expect the services to be restarted to reflect the change?